internal/extensions/extensions_test.go

```diff
@@ -0,0 +1,148 @@
+package extensions
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"testing"
+
+	_ "github.com/mattn/go-sqlite3"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidateExternalExtension(t *testing.T) {
+	cases := []struct {
+		name      string
+		extension string
+		wantError bool
+	}{
+		{"Valid Extension", "valid_extension", false},
+		{"Empty Extension", "", true},
+		{"Invalid Characters", "Invalid-Extension!", true},
+		{"Invalid Start Underscore", "_invalidextension", true},
+		{"Invalid End Underscore", "invalidextension_", true},
+		{"Invalid Non Alphanumeric", "invalid-extension", true},
+		{"Invalid Start Numeric", "123_invalid", false},
```

Comment:
`wantError` is set to `false` here so should the test name say `Valid Start Numeric` instead?

Reply:
yes! thanks for seeing this

Reply:
fixed

Reply:
fixed